xDrip/Services/CloudSync/NightscoutService.swift

```diff
@@ -18,28 +19,53 @@ final class NightscoutService {
     private var requestQueue: [UploadRequest] = []
     private var isRequestInProgress = false
     
-    private lazy var settingsObservers: [Any] = NotificationCenter.default.subscribe(
-        forSettingsChange: [.followerAuthStatus]) {
-            guard let settings = User.current.settings.nightscoutSync else { return }
-            if settings.isFollowerAuthed {
-                self.startFetchingFollowerData()
-            } else {
-                self.stopFetchingFollowerData()
-            }
+    private lazy var followerSettingsObserver: [Any] = NotificationCenter.default.subscribe(
+    forSettingsChange: [.followerAuthStatus]) {
```

Comment:
Code style. This line should have an additional indentation, so it is clear this line is a part of previous declaration.